readthedocs/ssh/restapi/urls.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+URLs for SSH key listing over API.
+
+.. note::
+
+    To define these URLs under a project, this line has to be included in a
+   ``urlpatterns`` loaded at start up:
+
+        url(r'^api/v2/', include('readthedocs.ssh.restapi.urls')),
+"""
+
+from __future__ import division, print_function, unicode_literals
+
+
+from django.conf.urls import url, include
+from rest_framework import routers
+
+from .model_views import ProjectSSHKeyViewSet
+
+# HACK: define new API endpoints under an existing path registered by DRF at
+# ``/project/<pk>/``. Instead of re-defining all of the views/endpoints we
+# filter the URLs returned by the router and only add the ones we want.
```

Comment:
Is there a good reason to do this here? Why not use the main `urls.py` and add a block like:

```python
if 'readthedocs.ssh' in settings.INSTALLED_APPS:
    # Add to urlconf
```